lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -90,6 +103,9 @@
   </style>
 </template>
 
+<!-- Force FCP to be ~5s out, necessary to make sure our render-blocking audits still work -->
+<!-- even when other forces were responsible for delaying the render. -->
+<script> while (Date.now() - performance.timing.responseStart < 5000) ; </script>
```

Comment:
can we use an add'l stylesheet with delay=5000 instead? Seems more natural.

Reply:
unfortunately, no. since the only way we assert that we didn't flag the amp stylesheet is by asserting the overall render-blocking stylesheets value is <3s. in the future details, but extendedInfo-style world this will be easier :)